source/includes/steps-repo.yaml

```diff
@@ -133,7 +133,22 @@ content: |
   .. code-block:: sh
 
      git remote add upstream https://github.com/<repoOwner>/<remoteRepoRoot>.git
+---
+title: Remove the redirects file
+ref: remove-redirects
+content: |
+
+  Some of our repositories have a ``redirects`` file that redirects browsers permanently to a new location.
+
+  Because non-admin users do not have permission to create or modify redirects, please remove the file.
```

Comment:
Removing this file is a good idea. However, I think we need a formal discussion about a process around redirect edits in the leads or other meeting. Once decided, this should be documented and circulated to the team.

Having the platform team approve root and certain redirect changes is a good thing. Requiring the platform team to review redirects on a page level (e.g., I just deleted a page and don't want to 404 our users until) may not be the best idea.

Reply:
Regarding language: 
- I suggest not starting with "Because"
- Who are the admin users? I think as a team we need to define this.
- Suggest: "The file must be removed because only admins have permission to create and modify redirects."

Reply:
> Removing this file is a good idea. However, I think we need a formal discussion about a process around redirect edits in the leads or other meeting. Once decided, this should be documented and circulated to the team.
> 
> Having the platform team approve root and certain redirect changes is a good thing. Requiring the platform team to review redirects on a page level (e.g., I just deleted a page and don't want to 404 our users until) may not be the best idea.

We have no way to prevent users from inadvertently redirecting our docs root and until we do we have to restrict this. We will create a way to make local changes truly local, we just don't have the bandwidth. It is my responsibility to prevent outages and I take that very seriously.